css/css-values/getComputedStyle-calc-mixed-units-003.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Values Test: computed value of calc() values using multiplication/division with mixed units</title>
+
+  <link rel="help" href="https://www.w3.org/TR/css-values-4/#calc-computed-value">
+
+  <meta name="flags" content="">
+  <meta content="This meta-test checks for the resolution and absolutization of computed values with mixed units to 'px' when multiplication/division is used." name="assert">
+
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+
+  <style>
+  html {
+      font-size: 30px;
+  }
+
+  body {
+      font-size: 16px;
+      line-height: 1.25; /* computed value: 20px */
+      width: 520px;
+      height: 500px;
+      margin: 20px;
+  }
+
+  div#target {
+      height: 100px;
+      width: 100px;
+  }
+  </style>
+
+  <div id="target"></div>
+
+  <script>
+  function startTesting() {
+    var targetElement = document.getElementById("target");
+
+    function verifyComputedStyle(property_name, specified_value, expected_value, description) {
+      test(function() {
+        targetElement.style.setProperty(property_name, "initial");
+        targetElement.style.setProperty(property_name, specified_value);
+        assert_equals(getComputedStyle(targetElement)[property_name], expected_value);
+      }, description);
```

Comment:
Nit: instead of passing `description` as an argument, you could do

```js
const description = `testing ${property_name}: {specified_value}`;
```

Reply:
Thanks, I'll make this change across other files as well to keep them uniform

Reply:
Thanks!